modules/mod_seo/templates/_html_head.tpl

```diff
@@ -46,7 +46,7 @@
 	<script>
 		window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
 		{% if m.acl.user %}
-			ga('create', "{{ ga }}", 'auto', { userId: m.acl.user});
+			ga('create', "{{ ga }}", 'auto', { userId: {{ m.acl.user }} });
```

Comment:
`m.acl.user` needs quoting otherwise it'll cause a javascript error. It's wise to also apply the `escapejs` filter. The entire line would become:

```
ga('create', "{{ ga|escapejs }}", 'auto', { userId: "{{ m.acl.user|escapejs }}" });
```